src/config/syscheck-config.c

```diff
@@ -824,21 +866,27 @@ char *syscheck_opts2str(char *buf, int buflen, int opts) {
         CHECK_SIZE,
         CHECK_OWNER,
         CHECK_GROUP,
-	CHECK_MD5SUM,
+        CHECK_MD5SUM,
         CHECK_SHA1SUM,
         CHECK_REALTIME,
         CHECK_SEECHANGES,
+#ifdef LIBSODIUM_ENABLED
```

Comment:
Having MD5 and SHA1 together makes some sense as both can have collisions, but dual-mode collisions are much less likely. SHA256 on the other hand is already computationally expensive, and not determined to have predictable collisions far as i know. Would it make more sense to
```
#ifdef LIBSODIUM_ENABLED
CHECK_SHA256SUM,
#else
CHECK_MD5SUM,
CHECK_SHA1SUM,
#end
```
so as to avoid the redundancy in processing and storage?

Reply:
I'm not against that, I was just continuing the tradition of using 2 hashes. I hope to replace md5 with blake2b (or whatever libsodium has as default at that point) in the future. md5 and sha256 give an analyst more opportunities to compare the hashes to "known bad" files out there. I've seen plenty of Indicators of Compromise reports still using md5.

Reply:
Good point, and some still use sha1, which sounds like "we might want them all." Maybe a semantic in which we store one hash and create other-algo-derived hashes on a change event for the alert would address that? Users could configure generation of event hashes in the fmt used by their malware engine, or just to pass the unk file to viper or whatever they're using for sample management to let another component deal with that rabbit hole.